docs/help-center/your-account/account-settings.mdx

```diff
@@ -489,6 +489,31 @@ Depending on the state of the chat backup, you have two options:
 
 In both cases, make sure to store the generated passphrase in a secure place.
 
+## Chat settings
+
+You can alter the press Enter action in this section.
+
+<Tabs>
+  <TabItem value="web" label="Web" default>
+  <ol>
+    <li>To access the settings, click on your profile picture or name initials.</li>
+    <li>Click on <b>Chat settings</b>.</li>
+    <li>By default, pressing Enter will send a new message.</li>
+    <li>If you want to start a new line by pressing Enter, click on <b>Start a new line</b> option</li>
+    <li>If you want to send a new message by pressing Enter, click on <b>Send message</b> option</li>
+  </ol>
+  </TabItem>
+  <TabItem value="mobile" label="Mobile">
+  <ol>
+    <li>To access the settings, tap on your profile picture or name initials.</li>
+    <li>Tap on <b>Chat settings</b>.</li>
+    <li>By default, tapping Enter will start a new line.</li>
+    <li>If you want to start a new line by tapping Enter, tap on <b>Start a new line</b> option</li>
```

Comment:
According to Figma it should be "Start new line"

Reply:
Fixed.